Kommunicate/Classes/KMConversationListViewController.swift

```diff
@@ -0,0 +1,642 @@
+            topVC.unsubscribingChannel()
+            topVC.viewModel.contactId = viewModel.contactId
+            topVC.viewModel.channelKey = viewModel.channelKey
+            topVC.viewModel.conversationProxy = viewModel.conversationProxy
+            topVC.viewWillLoadFromTappingOnNotification()
+            topVC.refreshViewController()
+        } else {
+            // push conversation VC
+            conversationVC.viewWillLoadFromTappingOnNotification()
+            navigationController?.pushViewController(conversationVC, animated: false)
+        }
+    }
+
+    private func showAlert(viewController:KMConversationListViewController) {
+        let alertMessage = LocalizedText.unableToCreateConversationError
+        let okText = LocalizedText.okButton
+        let alert = UIAlertController(
+            title: "",
+            message: alertMessage,
+            preferredStyle: UIAlertController.Style.alert)
+        alert.addAction(UIAlertAction(title: okText, style: UIAlertAction.Style.default, handler: nil))
+        viewController.present(alert, animated: true, completion: nil)
+    }
+
+    private func createConversationAndLaunch(notification:Notification){
+
+        guard let vc = notification.object as? KMConversationListViewController else {
+            return
+        }
+        vc.view.isUserInteractionEnabled = false
+        vc.navigationController?.view.isUserInteractionEnabled = false
+        let alertView =  displayAlert(viewController : vc)
+
+        Kommunicate.createConversation() { (result) in
+            switch result {
+            case .success(let conversationId):
+                DispatchQueue.main.async {
+                    vc.view.isUserInteractionEnabled = true
+                    vc.navigationController?.view.isUserInteractionEnabled = true
+                    alertView.dismiss(animated: false, completion: nil)
+                    Kommunicate.showConversationWith(groupId: conversationId, from: vc, completionHandler: { (success) in
+                        print("Conversation was shown")
+                    })
+                }
+            case .failure( _):
+                DispatchQueue.main.async {
+                    vc.view.isUserInteractionEnabled = true
+                    vc.navigationController?.view.isUserInteractionEnabled = true
+                    alertView.dismiss(animated: false, completion: {
+                        self.showAlert(viewController: vc)
+                    })
+                }
+            }
+        }
+    }
+
+    private func displayAlert(viewController:KMConversationListViewController) -> UIAlertController {
+
+        let alertTitle = LocalizedText.waitMessage
+        let loadingAlertController = UIAlertController(title: alertTitle, message: nil, preferredStyle: .alert)
+
+        let activityIndicator: UIActivityIndicatorView = UIActivityIndicatorView(style: .gray)
+        activityIndicator.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
We have already defined an `activityIndicator` at the top, so same can be used here.